app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -99,7 +99,7 @@ class AdministratorControlsActivityTest {
   @Test
   // TODO(#973): Fix AdministratorControlsActivityTest
   @Ignore
-  fun testAdministratorControlsActivity_withAdminProfile_openAdministratorControlsActivityFromNavigationDrawer_onBackPressed_showsHomeActivity() { // ktlint-disable max-line-length
+  fun testAdminCtrls_withAdminProfile_openAdminCtrlsFromNavDrawer_onBackPressed_showsHome() {
```

Comment:
Here & elsewhere: you could also use ``testActivity_...`` or ``testFragment_...`` (consider that the test name also helps contextualize these names).

Reply:
Got it @BenHenning , changed it where it helped!